spec/support/active_storage.rb

```diff
@@ -1,6 +1,6 @@
 RSpec.configure do |config|
   config.before(:each) do
-    ActiveStorage::Current.host = 's3.example.com'
+    Rails.application.routes.default_url_options[:host] = 's3.example.com'
```

Comment:
This sets the default host to `s3.example.com` everywhere - so, for example, if we sent an email in a test  links would contain `s3.example.com` as the host, which is not correct.

`ActiveStorage::Current.host` sets the host for S3 only, but for some reason this isn't working after Rails v5.2.4.